image/autodetect.go

```diff
@@ -67,46 +66,11 @@ func Autodetect(path string) (string, error) {
 		return TypeImage, nil
 
 	case "text/plain; charset=utf-8":
```

Comment:
Uh，This is the original code, the results by `http.DetectContentType` to determine the format.